src/explore-education-statistics-admin/src/pages/methodology/createMethodologyPage.tsx

```diff
@@ -0,0 +1,223 @@
+  dayMonthYearValuesToInputs,
+  dayMonthYearInputsToDate,
+} from '@common/services/publicationService';
+import { validateMandatoryDayMonthYearField } from '@admin/validation/validation';
+
+interface MatchProps {
+  topicId: string;
+}
+
+interface FormValues {
+  methodologyTitle: string;
+  selectedContactId: string;
+  scheduledPublishDate: DayMonthYearInputs;
+}
+
+const serverSideValidationHandler = handleServerSideValidation(
+  errorCodeToFieldError(
+    'SLUG_NOT_UNIQUE',
+    'methodologyTitle',
+    'Choose a unique title',
+  ),
+);
+
+interface CreateMethodologyModel {
+  contacts: ContactDetails[];
+}
+
+const CreateMethodologyPage = ({
+  match,
+  history,
+}: RouteComponentProps<MatchProps>) => {
+  const [model, setModel] = useState<CreateMethodologyModel>();
+
+  useEffect(() => {
+    Promise.all([service.getPublicationAndReleaseContacts()]).then(
+      ([contacts]) => {
+        setModel({
+          contacts,
+        });
+      },
+    );
+  }, []);
+
+  const submitFormHandler = async (values: FormValues) => {
+    const submission: CreateMethodologyRequest = {
+      title: values.methodologyTitle,
+      publishScheduled: dayMonthYearInputsToDate(values.scheduledPublishDate),
+      contactId: values.selectedContactId,
+    };
+
+    const createdMethodology = await methodologyService.createMethodology(
+      submission,
+    );
+
+    // TODO: redirect to the methodology summary
+    history.push(`/methodology/${createdMethodology.id}`);
+  };
+
+  const cancelHandler = () => history.push('/methodology');
+
+  const getSelectedContact = (
+    contactId: string,
+    availableContacts: ContactDetails[],
+  ) =>
+    availableContacts.find(contact => contact.id === contactId) ||
+    availableContacts[0];
+
+  const formId = 'createMethodologyForm';
+
+  return (
+    <Page
+      wide
+      breadcrumbs={[
+        {
+          name: 'Manage methodology',
+          link: '/methodology',
+        },
+        {
+          name: 'Create new methodology',
+        },
+      ]}
+    >
+      <div className="govuk-grid-row">
+        <div className="govuk-grid-column-two-thirds">
+          <h1 className="govuk-heading-xl">Create new methodology</h1>
+        </div>
+        <div className="govuk-grid-column-one-third">
+          <RelatedInformation heading="Help and guidance">
+            <ul className="govuk-list">
+              <li>
+                <Link to="/documentation" target="blank">
+                  Creating new methodology{' '}
+                </Link>
+              </li>
+            </ul>
+          </RelatedInformation>
+        </div>
+      </div>
+      {model && (
+        <Formik<FormValues>
+          enableReinitialize
+          initialValues={{
+            methodologyTitle: '',
+            selectedContactId: orderBy(
+              model.contacts,
+              contact => contact.contactName,
+            )[0].id,
+            scheduledPublishDate: dayMonthYearValuesToInputs(
```

Comment:
Are we doing anything with schedulePublished, cos most of it is commented out in the backend

Reply:
See comment above :)